.github/workflows/deployment-images.yaml

```diff
@@ -5,6 +5,18 @@ on:
       build-information-path:
         required: true
         type: string
+      docker-build-dir:
+        required: false
+        type: string
+        default: '.'
+      docker-build-file:
```

Comment:
The number of parameters can be minimized by adopting a "convention over configuration" approach. Is it not possible for the main `Dockerfile` for your project(s) to be named `Dockerfile`?

Reply:
Yes, if the Dockerfile is only one, but in case with more they have to be named differently. Look at the example projects I gave in the issue section https://github.com/rentpath/actions-services-workflows/issues/6